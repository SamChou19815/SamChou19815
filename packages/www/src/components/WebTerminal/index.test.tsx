import React from 'react';
import renderer from 'react-test-renderer';
import { WebTerminal } from '.';

// eslint-disable-next-line @typescript-eslint/no-empty-function
const ignore = () => {};

it(`WebTerminal(shown) matches snapshot.`, () => {
  const tree = renderer.create(<WebTerminal shown onTrigger={ignore} />).toJSON();
  expect(tree).toMatchSnapshot();
});

it(`WebTerminal(hidden) matches snapshot.`, () => {
  const tree = renderer.create(<WebTerminal shown={false} onTrigger={ignore} />).toJSON();
  expect(tree).toMatchSnapshot();
});
