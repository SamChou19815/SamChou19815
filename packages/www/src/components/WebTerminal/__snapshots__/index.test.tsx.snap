// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WebTerminal(hidden) matches snapshot. 1`] = `
<button
  className="TriggerButton"
  onClick={[Function]}
  type="button"
>
  <span
    className="FirstEdge"
  />
  <span
    className="SecondEdge"
  />
</button>
`;

exports[`WebTerminal(shown) matches snapshot. 1`] = `
Array [
  <button
    className="TriggerButton"
    onClick={[Function]}
    type="button"
  >
    <span
      className="FirstEdge"
    />
    <span
      className="SecondEdge"
    />
  </button>,
  <div
    className="TerminalContainer"
  >
    <div
      className="TerminalTitle"
    >
      Terminal
    </div>
    <div
      className="Terminal"
      onClick={[Function]}
      role="presentation"
    >
      <div
        className="TerminalContent"
      >
        <p
          className="TerminalMessage"
        >
          Type \`help\` to show a list of available commands.
        </p>
        <div
          className="TerminalInputArea"
        >
          <span
            className="TerminalPromptLabel"
          >
            $
          </span>
          <input
            autoComplete="off"
            className="TerminalInput"
            name="terminal-input"
            onChange={[Function]}
            onKeyDown={[Function]}
            type="text"
            value=""
          />
        </div>
      </div>
    </div>
  </div>,
]
`;
